nncf/pruning/filter_pruning/algo.py

```diff
@@ -128,8 +131,9 @@ def _apply_masks(self):
 
         def _apply_binary_mask_to_module_weight_and_bias(module, mask, module_name=""):
             with torch.no_grad():
+                dim = module.target_weight_dim_for_compression if isinstance(module, _NNCFModuleMixin) else 0
```

Comment:
Why do you need here this check?
Not all pruned ops are subclasses of `_NNCFModuleMixin`?

Reply:
Yes, it's need for BatchNorm.